packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/signup-code.js

```diff
@@ -27,6 +27,11 @@ const ROLLOUT_CLIENTS = {
     name: 'Lockbox Extension iOS',
     rolloutRate: 0.0,
   },
+  e6eb0d1e856335fc: {
```

Comment:
Not for this PR of course, but would this feature be a good candidate for redis based feature flagging?